docker-compose.yml

```diff
@@ -40,12 +40,12 @@ services:
     networks:
       - io-spid-commons
 
-  spid-saml-check:
-    image: spid-saml-check
-    ports:
-      - "8080:8080"
-    networks:
-      - io-spid-commons
+  #spid-saml-check:
```

Comment:
why this change?

Reply:
`spid-saml-check` is no more available on docker-hub. Can we use [this](https://hub.docker.com/r/superdell/spid-saml-check) container?

Reply:
nope, it's better to leave this as-is and bake our own locally from github source code 
(we should add such a docker image into this repo probably)